### PR TITLE
Add Spanish version.

### PR DESCRIPTION
**Pull request by pazguille**
Done!

### CODE REVIEW

File: es/index.html [hunk 1]
```diff
@@ -0,0 +1,502 @@
...
+  </head>
+  <body>
+    <main>
+      <div class="content">
+        <div class="content">
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__list-item">
+              <a class="nav__link" href="./../index.html">
+                <img class="nav__image" src="./../images/flag-gb.png" alt="English" />
```
Comment (sergeicodes) [inline es/index.html]: @pazguille could you translate nav images `alt` text?

File: es/index.html [hunk 2]
```diff
@@ -0,0 +1,502 @@
...
+          <a target="_blank" href="https://en.wikipedia.org/wiki/Hand">Manos</a
+          >. Hosteado en
+          <a target="_blank" href="https://www.netlify.com/">Netlify</a>. El código
+          está en
+          <a target="_blank" href="https://github.com/sergeicodes/a11ymyths"
+            >GitHub</a
+          >. Seguime
+          <a target="_blank" href="https://twitter.com/_sergeikriger"
+            >@_sergeikriger</a
+          >. Translated by
```
Comment (sergeicodes) [inline es/index.html]: @pazguille could you translate `Translated by`?

File: es/index.html [hunk 3]
```diff
@@ -0,0 +1,502 @@
...
+          <a target="_blank" href="https://twitter.com/pazguille"
+            >@pazguille</a
+          >.
+        </p>
+        <p>
+          Icons made by
```
Comment (sergeicodes) [inline es/index.html]: @pazguille could you translate these icon credentials?

**Review by sergeicodes — CHANGES_REQUESTED** [review]

**Comment by sergeicodes** [comment]
@pazguille thanks for translating, like it!

File: es/index.html [hunk 4]
```diff
@@ -0,0 +1,502 @@
...
+            Bueno, por lo general las personas con alguna discapacidad son las mejores para probar
+            - ya que todo el tiempo usan tecnologías asistivas. Sin embargo,
+            <strong>cualquiera puede aprender</strong> a probar la accesibilidad
+            de los sitio webs.
+          </p>
+
+          <a href="https://www.w3.org/WAI/test-evaluate/" target="_blank"
+            >Más detalles
+            <span class="visually-hidden"
+              >sobre Accessibility can only be tested by disabled people</span
```
Comment (sergeicodes) [inline es/index.html]: @pazguille could you add translation for `Accessibility can only be tested by disabled people`?

**Comment by pazguille** [comment]
New translations added!

**Review by sergeicodes — APPROVED** [review]

**Pull request merged**